src/pages/product-listing/products-listing.tsx

```diff
@@ -0,0 +1,24 @@
+import { useContext } from "react";
+import ProductListingItem from "../../components/product-listing-item/product-listing-item";
+import { ProductContext } from "../../providers/products-provider";
+
+const ProductsListing = () => {
+  const { contextProducts:products, error, isLoading } = useContext(ProductContext);
+  return (
+    <div>
```

Comment:
You can avoid adding an extra node to the DOM and replace the <div> with the short syntax of React.Fragment. You should do this whenever possible.

Reply:
I kind of need the wrapper element in the way I designed the layout. I just forgot to assign it a class.
Besides that, all other components from `/pages` have a wrapper element for consistency.